### ISSUE TITLE
Showing status

### ISSUE THREAD

**Issue by govuk-design-system** [alphagov/govuk-design-system-backlog#98]
## What
Help users understand the status of something, so they can decide what action to perform next.

## Why
Statuses are used in many government services, especially internal ones. Many different styles are used however, so some guidance and good examples here would be useful.

## Anything else
- [Status indicator in Home Office Design System](https://github.com/UKHomeOffice/design-system/issues/53)
- [Status tracking discussion in Dropbox Paper](https://paper.dropbox.com/doc/Status-tracking-vMeDLJY7BwgXKpaCUMm2G)

## Related patterns
- #62 Tag
- #72 Task list

**Comment by joelanman** [comment]
Considering we have both Tag and Task list published, seems like it might be easy to get this one done

**Comment by christopherthomasdesign** [comment]
We’ve been looking at showing status in our work on managing the certificates that teams connected to GOV.UK Verify use for encryption.

We’ve basically just used something very similar to a task list for showing the status of each certificate:

<img width="660" alt="Screen Shot 2019-07-05 at 16 06 11" src="https://user-images.githubusercontent.com/19834460/60732508-1cb82980-9f42-11e9-89f1-07d61a716407.png">

The labels should indicate whether the user needs to do something. And should reflect the real, live status of the thing they represent.

For some certificates, there’s a period where a new certificate is being uploaded and the old one is still in use. We use this ‘deploying’ tag:

<img width="657" alt="Screen Shot 2019-07-05 at 16 08 05" src="https://user-images.githubusercontent.com/19834460/60732520-2477ce00-9f42-11e9-8459-fc145202eb97.png">

This status is temporary, and we found that users needed a little more clarification on what’s going on. So on the page for that certificate, we added a banner at the top of the linked page (styled like an error summary/alert but coloured to match the tag). This explains what is happening in more detail:

<img width="646" alt="Screen Shot 2019-07-05 at 16 19 05" src="https://user-images.githubusercontent.com/19834460/60732526-293c8200-9f42-11e9-9bf9-a5fa26bd22f7.png">

So basically it’s just a combination of slightly adapted task list / alert components. It’s taken a few iterations but this is beginning to test quite well, would be interested to see what others are doing.
